<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subwindow</class>
 <widget class="QWidget" name="subwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="newfilename">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="check">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="check1">
      <property name="text">
       <string>.h文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="check2">
      <property name="text">
       <string>.cpp文件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="check3">
      <property name="text">
       <string>硬件文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
